ihmc-high-level-behaviors/src/main/java/us/ihmc/PerceptionAndAutonomyProcess.java

```diff
@@ -20,15 +20,11 @@
 import us.ihmc.communication.ros2.ROS2PublishSubscribeAPI;
 import us.ihmc.euclid.referenceFrame.ReferenceFrame;
 import us.ihmc.log.LogTools;
-import us.ihmc.perception.BallDetectionManager;
-import us.ihmc.perception.BytedecoImage;
-import us.ihmc.perception.IterativeClosestPointManager;
-import us.ihmc.perception.RapidHeightMapManager;
-import us.ihmc.perception.RawImage;
+import us.ihmc.perception.*;
```

Comment:
Do you have the most up to date formatter? I was under the impression that we increased the number of imports before the wildcard would get used?